src/test/java/com/wnis/linkyway/controller/member/MemberControllerTest.java

```diff
@@ -0,0 +1,99 @@
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.http.MediaType;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(controllers = MemberController.class,
+        excludeFilters = { @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE,
+                classes = WebSecurityConfigurerAdapter.class) })
+public class MemberControllerTest {
+    
+    @Autowired
+    MockMvc mockMvc;
+    
+    @MockBean
+    MemberService memberService;
+    
+    @Autowired
+    WebApplicationContext ctx;
+    
+    @Autowired
+    ObjectMapper objectMapper;
+    
+    @BeforeEach
+    void setUp() {
+        mockMvc = MockMvcBuilders.webAppContextSetup(ctx)
+                                 .alwaysDo(print())
+                                 .build();
+    }
+    
+    
+    @Nested
+    @DisplayName("회원 수정")
+    class updateMemberTest {
+        
+        @Test
+        @DisplayName("입력 Validation 테스트")
+        void shouldReturnStatus400_WhenInvalidInputTest() throws Exception {
+            UpdateMemberRequest updateMemberRequest = UpdateMemberRequest.builder()
+                    .nickname("adksjflkajsdljaksdflksjflasdkfjsdjfjsalkjfasdfas").build();
+            
+            mockMvc.perform(put("/api/members/page/me")
+                                    .contentType(MediaType.APPLICATION_JSON)
+                                    .content(objectMapper.writeValueAsString(updateMemberRequest)))
+                   .andExpect(status().is(400));
+        }
+    
+        @Test
+        @DisplayName("닉네임 중복시 테스트")
+        void shouldReturnStatus409_WhenThrowDuplicateNicknameTest() throws Exception {
+            UpdateMemberRequest updateMemberRequest = UpdateMemberRequest.builder()
+                                                                         .nickname("hello").build();
+    
+            doThrow(new NotAddDuplicateEntityException("h")).when(memberService).updateMyPage(any(), any());
+            
+            mockMvc.perform(put("/api/members/page/me")
+                                    .contentType(MediaType.APPLICATION_JSON)
+                                    .content(objectMapper.writeValueAsString(updateMemberRequest)))
+                   .andExpect(status().is(409));
```

Comment:
```suggestion
                   .andExpect(status().isConflict());
```

이건 그냥 개인적인 생각인데 하드코딩하지말라고 제공해주는 애들을 사용하면 실수도 안하고 가독성도 좋아지지 않나 싶습니다

내부적으로 다음과 같은 애를 사용합니다
```java
CONFLICT(409, Series.CLIENT_ERROR, "Conflict")
```

Reply:
사실 어떻게 보면 이 부분 사실 에러코드 아는 개발자에겐 명확한 숫자이긴하지만 멘토님 리뷰 때 매직 넘버 말씀하신게 기억나네요.. 적용해볼께요..!